src/main/java/edu/kis/powp/command/OperateToCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class OperateToCommand implements DriverCommand{
+
+    private int x;
+    private int y;
+
+    public OperateToCommand(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    @Override
+    public void execute(DriverManager driverManager) {
+        driverManager.getCurrentDriver().operateTo(x, y);
```

Comment:
driverManager.getCurrentDriver() could be passed externally. Job2dDriver could be the param.